### PR TITLE
Daily Update

### PR DESCRIPTION
**Pull request by HaibunisCool**
- Updated user profile
- Trail Page
- Switched services to revature.Trailmates

### CODE REVIEW

**Comment by HaibunisCool** [comment]
- Also finished notifications

File: Trailmates/src/app/profile-page/trail-history/trail-history.component.html [hunk 1]
```diff
@@ -1,29 +1,41 @@
+<<<<<<< HEAD
```
Comment (MattB120) [inline Trailmates/src/app/profile-page/trail-history/trail-history.component.html:1]: Merge conflict remains

File: Trailmates/src/app/profile-page/trail-history/trail-history.component.html [hunk 2]
```diff
@@ -1,29 +1,41 @@
...
-                    <input type="file" id="myFile" name="filename">
+                    <input type="file" id="myFile" accept="image/*" name="filename">
             </div>
         </div>
+=======
+<div class ="container">
+    <button class="close" (click)="close()" >&times;</button>
+    <h1>Create a New Post</h1>
+    
+>>>>>>> f5479050a005972bca3c7e5891ab31b41bab571b
```
Comment (MattB120) [inline Trailmates/src/app/profile-page/trail-history/trail-history.component.html:15]: Merge conflict remains

File: Trailmates/src/app/services/friend.service.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import { User } from '../models/user';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class FriendService {
+
+  constructor(private http: HttpClient) { }
+  url: string = 'http://trailmates-env.us-east-1.elasticbeanstalk.com/TrailMates/friends/'
```
Comment (MattB120) [inline Trailmates/src/app/services/friend.service.ts:16]: Still old url should be```https://revature.trailmates.net/TrailMates/```

**Review by MattB120 — CHANGES_REQUESTED** [review]

**Review by labrecquemegan — APPROVED** [review]

**Review by MattB120 — APPROVED** [review]

**Pull request merged**